src/index.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <div>Hello, world!</div>,
```

Comment:
It might be a good idea to keep `<React.StrictMode>` - it just adds more useful warnings/errors in the console as you're developing. But it's fine to remove it if you don't find the messages helpful.